apps/calendar/js/views/single_month.js

```diff
@@ -32,6 +32,7 @@ SingleMonth.prototype = {
   element: null,
 
   create: function() {
+    startDay = Calc.startDay;
```

Comment:
so we should remove it from the other line.. because the `Calc.startDay` value can change during locale change and the months are rebuilt every time the locale changes. (that's why we update the value during `create`)